webpack.config.js

```diff
@@ -15,8 +15,8 @@ module.exports = {
     umdNamedDefine: true
   },
   externals: {
-    'simplemde': 'SimpleMDE',
-    'marked': 'Marked'
+    'simplemde': 'simplemde',
```

Comment:
I think this line should be `'simplemde': 'SimpleMDE',`.

Reply:
``` html
<script src="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.js"></script>
```
``` javascript
<script>
var simplemde = new SimpleMDE();
</script>
```

Reply:
I tried that and npm warned that cannot resolve 'Marked' and 'SimpldeMDE'. Lower case version compiled without any errors

Reply:
emmmm...sorry, my mistake. we can use an object to describe all the possible ways. ([doc](https://webpack.js.org/configuration/externals/))
``` javascript
externals : {
  simplemde : {
    commonjs: 'simplemde',
    amd: 'simplemde',
    root: 'SimpleMDE' 
  },
  'marked': 'marked'
}
```

Reply:
If there is no problem, please update and bump package version to `0.4.6` .
Thanks.